mbus-api/src/message_bus/v0.rs

```diff
@@ -168,6 +202,113 @@ pub trait MessageBusTrait: Sized {
         let _ = request.request().await?;
         Ok(())
     }
+
+    /// Get nexuses with filter
+    #[tracing::instrument(level = "debug", err)]
+    async fn get_nexuses(filter: Filter) -> BusResult<Vec<Nexus>> {
+        let nexuses = GetNexuses {
+            filter,
+        }
+        .request()
+        .await?;
+        Ok(nexuses.0)
+    }
+
+    /// Get nexus with filter
+    #[tracing::instrument(level = "debug", err)]
+    async fn get_nexus(filter: Filter) -> BusResult<Nexus> {
+        let nexuses = Self::get_nexuses(filter).await?;
+        only_one!(nexuses)
+    }
+
+    /// create nexus
+    #[tracing::instrument(level = "debug", err)]
+    async fn create_nexus(request: CreateNexus) -> BusResult<Nexus> {
+        let nexus = request.request().await?;
+        Ok(nexus)
+    }
+
+    /// destroy nexus
+    #[tracing::instrument(level = "debug", err)]
+    async fn destroy_nexus(request: DestroyNexus) -> BusResult<()> {
+        request.request().await?;
+        Ok(())
+    }
+
+    /// share nexus
+    #[tracing::instrument(level = "debug", err)]
+    async fn share_nexus(request: ShareNexus) -> BusResult<String> {
+        let share = request.request().await?;
+        Ok(share)
```

Comment:
This should be written as request.request.await without the explicit `?;` and `Ok(share)` The same goes for most lines where we have this format or Ok(request.wait?) formats.